### ISSUE TITLE
SyntaxError: Unexpected token ILLEGAL

### ISSUE THREAD

**Issue by janl** [finnp/tito-gender-ratio#1]
```
> tito-gender-ratio < tito.csv 

/usr/local/lib/node_modules/tito-gender-ratio/cli.js:15
    console.log(`${Math.round(r.female / (r.male + r.female + r.missing) * 100
                ^
SyntaxError: Unexpected token ILLEGAL
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:929:3

```

**Comment by finnp** [comment]
Thanks for the report :) This should be resolved in 2.0.1.  :ant:

**Comment by janl** [comment]
works, thanks!
